app/controllers/account/device_groups_controller.rb

```diff
@@ -0,0 +1,64 @@
+class Account::DeviceGroupsController < Account::DashboardController
+  def index
+    @device_groups = collection
+  end
+
+  def new
+    @device_group = DeviceGroup.new
+  end
+
+  def create
+    @device_group = parent.device_groups.build(device_group_params)
+    if @device_group.save
+      flash[:notice] = 'Group saved!'
+      redirect_to account_device_groups_path
+    else
+      flash[:error] = 'Failed to create a group!'
```

Comment:
ensure to use `flash.now` for cases if this message wouldn't be displayed as it may be stored in session during several redirects